light-client/eth-light-client/src/client.rs

```diff
@@ -0,0 +1,67 @@
+use std::{
+	path::PathBuf,
+	process::exit,
+	sync::{
+		atomic::{AtomicBool, Ordering},
+		Arc,
+	},
+};
+
+use ethers::types::Address;
+use eyre::Result;
+use helios::{client::ClientBuilder, config::networks::Network, prelude::*};
+
+use crate::{config::Config, db::DB};
+
+pub async fn start_client(config: Config, db: DB, term: Arc<AtomicBool>) -> Result<()> {
+	let mut client: Client<FileDB> = ClientBuilder::new()
+		.network(Network::MAINNET)
+		.consensus_rpc(&config.consensus_rpc)
+		.execution_rpc(&config.untrusted_rpc)
+		.load_external_fallback()
+		.data_dir(PathBuf::from(
+			config.helios_home_path.unwrap_or("/tmp/helios".to_string()),
+		))
+		.build()?;
+
+	log::info!(
+		"Built client on network \"{}\" with external checkpoint fallbacks",
+		Network::MAINNET
+	);
+
+	exit_if_term(term.clone());
+
+	client.start().await?;
+	log::info!("client started");
+
+	let filter = ethers::types::Filter::new()
+		.select(
+			config
+				.block_number
+				.map(Into::into)
+				.unwrap_or(ethers::core::types::BlockNumber::Latest)..,
+		)
+		.address(config.smart_contract_address.parse::<Address>()?)
+		.event("Transfer(address,address,uint256)");
+
+	loop {
+		exit_if_term(term.clone());
+		let logs = client.get_logs(&filter).await?;
+		log::info!("logs: {:#?}", logs);
```

Comment:
probably it would be better to dump it as debug.

Also, it would be nice to add target: "client", target: "server" for different logs

Reply:
Fixed.